spec.bs

```diff
@@ -1493,8 +1493,10 @@ a {{DirectFromSellerSignalsForBuyer}} |directFromSellerSignalsForBuyer|, a [=dur
     1. [=map/Set=] |prevWinIDL|["{{PreviousWin/adJSON}}"] to |prevWin|'s [=previous win/ad json=].
     1. [=list/Append=] |prevWinIDL| to |prevWins|.
   1. [=map/Set=] |browserSignals|["{{BiddingBrowserSignals/prevWinsMs}}"] to |prevWins|.
-  1. Let « |biddingScript|, <var ignore>ignored</var> » be the result of [=fetching script=] with
+  1. Let |biddingScriptFetcher| be the result of [=creating a new script fetcher=] with
     |ig|'s [=interest group/bidding url=].
+  1. Let |biddingScript| be the result of [=waiting for script body from a fetcher=] given

```

Comment:
Is this the right way to describe a non-blocking wait on a fetch?  Reading the stuff this and the above line link to, it sounds like we queue a task off thread...and then synchronously wait for it to complete on the main thread.

Reply:
You're correct --- it doesn't change anything at this call sight [1].  Seller scripts are improved to:
1) Start one fetch early, not have one every time we want to call scoreAd() (!)
2) Have some asynchrony/overlap in timing between the fetch start and use.
... the plan to get both of those for bidders is to basically make a giant map and fetch all the scripts, as a separate change. 

...but to fully fix parallelism, we need to make the actual bidding happen all in parallel, too, so the blocking happens off-thread. Which is tricky because of shared data structures.

My immediate interest was in making the dependency on the headers for cross-origin trusted seller signals stuff more explicit.

Reply:
>...but to fully fix parallelism, 

So the implementation has parallelism, the spec just isn't adequately describing it?